e2e/common/operator_metrics_test.go

```diff
@@ -171,6 +172,17 @@ func TestMetrics(t *testing.T) {
 			))
 		})
 
+		/*
+		 * TODO
+		 * The duration_seconds tests keep randomly failing on OCP4 with slightly different duration values
+		 * May need to lessen the strict checking parameters
+		 *
+		 * Adding CAMEL_K_TEST_SKIP_PROBLEMATIC env var for the moment.
+		 */
+		if os.Getenv("CAMEL_K_TEST_SKIP_PROBLEMATIC") == "true" {
```

Comment:
Could the problematic tests be deactivated only on OCP4? Most of them run fine on other environments, so it'd be better not to disable them globally in the interim.

Reply:
That is the case. Only in the nightly OCP test suite is CAMEL_K_TEST_SKIP_PROBLEMATIC set to "true". Upstream is empty.

Reply:
Ah, I checked the Kubernetes workflow logs and saw:
```
=== Problematic Tests (7) ===
  ./e2e/common/cli/dev_mode_test.go::TestRunDevMode
  ./e2e/common/cli/dev_mode_test.go::TestRunDevMode
  ./e2e/common/operator_metrics_test.go::TestMetrics
  ./e2e/common/operator_metrics_test.go::TestMetrics
  ./e2e/common/traits/health_test.go::TestHealthTrait
  ./e2e/common/traits/toleration_test.go::TestTolerationTrait
  ./e2e/common/traits/route_test.go::TestRunRoutes
===
```
I assumed the tests got skipped as a result, but they are indeed executed later on. Maybe the problematic tests listing could be skipped when `CAMEL_K_TEST_SKIP_PROBLEMATIC` is not truthy.